paasta_tools/cli/cmds/local_run.py

```diff
@@ -622,6 +622,8 @@ def get_local_run_environment_vars(instance_config, port0, framework):
         env["MARATHON_APP_LABELS"] = ""
         env["MARATHON_APP_ID"] = "/simulated_marathon_app_id"
         env["MARATHON_HOST"] = hostname
+        env["PAASTA_HOST"] = hostname
```

Comment:
++

Reply:
I added PAASTA_GIT_SHA, which parses docker url to get the git-sha using 
 def get_code_sha_from_dockerurl(docker_url: str) -> str:
```
     """We encode the sha of the code that built a docker image *in* the docker
     url. This function takes that url as input and outputs the partial sha
     """
     try:
         parts = docker_url.split("/")
         parts = parts[-1].split("-")
         return "git%s" % parts[-1][:8]
     except Exception:
         return "gitUNKNOWN"
```

It's hard to add container id since we cannot know it before creating the container with docker_run.

Reply:
> It's hard to add container id since we cannot know it before creating the container with docker_run.

Oh yeah good point, didn't think of that. Just ignore it then